modules/administration-guide/pages/defining-the-list-of-images-to-pull.adoc

```diff
@@ -7,39 +7,16 @@
 
 The {image-puller-name-short} can pre-pull most images, including scratch images such as `che-machine-exec`. However, images that mount volumes in the Dockerfile, such as `traefik`, are not supported for pre-pulling on OpenShift 3.11.
 
-Pre-pulling images involved in workspace startup will reduce workspace start times. For example:
-
-* Che-Theia
-* broker images
-* plug-in sidecar images
-
-.Prerequisites
-
-* The `curl` tool is available. See link:https://curl.se/[curl homepage].
-* The `jq` tool is available. See link:https://stedolan.github.io/jq/[jq homepage].
-* The `yq` tool is available. See link:https://pypi.org/project/yq/[yq homepage].
-
 .Procedure
 
-. Gather a list of relevant container images for the {platforms-name} platform:
-+
-include::example$snip_{project-context}-getting-the-list-of-relevant-images.adoc[]
-
+. Gather a list of relevant container images for prepulling by navigating to the `pass:c,a,q[{prod-url}]/plugin-registry/v3/external_images.txt`
```

Comment:
```suggestion
. Gather a list of relevant container images for pre-pulling by navigating to the `pass:c,a,q[{prod-url}]/plugin-registry/v3/external_images.txt` file.
```

Reply:
well, technically this is a URL, not a file
@Eva-Lotte should we use URL instead?

Reply:
fixed